src/main/webapp/app/shared/user-settings/ide-preferences/ide-settings.scss

```diff
@@ -0,0 +1,32 @@
+h3 {
+    font-weight: bold;
+    text-underline-offset: 3px;
+}
+
+dt {
+    font-size: large;
+    font-weight: 500;
+}
+
+dd {
+    font-size: larger;
+    color: gray;
```

Comment:
Please check the theming guidelines for handling colors and use a variable (`var(-- ...)`) here as described and make sure it looks good in light and dark mode

https://docs.artemis.cit.tum.de/dev/guidelines/client-design.html